### ISSUE TITLE
Networking tab interface table link speed unimplemented

### ISSUE THREAD

**Issue by jondanb** [jondanb/classic-windows-task-manager-linux#14]
```py
def update_networking_page_net_list_table(
        self, interface_full_name, net_utilization="-", link_speed="-", network_state="-"):
        self.append_row_to_table(
            self.parent.net_t_network_list_table, CWTM_NetworkingTabTableColumns,
            CWTM_TableWidgetItemProperties(item_label=interface_full_name, 
                item_tool_tip=interface_full_name),
            CWTM_TableWidgetItemProperties(item_label=net_utilization), 
            CWTM_TableWidgetItemProperties(item_label=link_speed), #todo
            CWTM_TableWidgetItemProperties(item_label=network_state)
        )
```
`link_speed` is left at its default value, there's no information retrieval function for it

**Comment by jondanb** [comment]
`nmcli -f CAPABILITIES.SPEED dev show <interface>` works but is a bash command
maybe there's a way to do this in code without using a subprocess?

**Comment by jondanb** [comment]
> `nmcli -f CAPABILITIES.SPEED dev show <interface>` works but is a bash command maybe there's a way to do this in code without using a subprocess?

Fixed using nmcli in 7e1e7eda46d7469e220fb64b8a89cdf0886964ab
